### ISSUE TITLE
Druid Feral - Ferocious Bite Doenst Work

### ISSUE THREAD

**Issue by buenodart** [ottconnect/Legion#124]
The skill ferocious bite doesnt work.

You get fury points and try to use the skill on a mob. Nothing happens.

I dont know if this is the way to open a ticket, if Im wrong, please tell my what to do.

Thanks

**Comment by LeRutY** [comment]
Yes, the combo point system (rogues and feral druids use that) is buggy. So, spells that spend points don´t work well because of that. Both things must be functional. It doesn´t make sense playing a rogue at the moment.

https://github.com/ottconnect/Legion/issues/42 "hansleo21" already reported the the combo point and stealth mechanics issues.

**Comment by ottconnect** [comment]
so this is a duplicate ticket?

**Comment by LeRutY** [comment]
Not really. Mainly it´s the spell, not the system that doesn´t work.
